src/go/rpk/pkg/cli/debug/bundle/bundle_k8s_linux.go

```diff
@@ -135,13 +137,28 @@ func executeK8SBundle(ctx context.Context, bp bundleParams) error {
 		if err != nil {
 			errs = multierror.Append(errs, err)
 		}
+		errs.ErrorFormat = errorFormat
 		fmt.Println(errs.Error())
 	}
 
 	fmt.Printf("Debug bundle saved to %q\n", f.Name())
 	return nil
 }
 
+// adminAddressesUnion returns the union of two slices of adminAddresses.
+func adminAddressesUnion(a, b []string) []string {
+	m := make(map[string]bool) // track unique addresses.
+	for _, v := range a {
+		m[v] = true
+	}
+	for _, v := range b {
+		if _, ok := m[v]; !ok {
```

Comment:
```suggestion
		if !m[v] {
```

The primary benefit of using `map[string]bool` (IMO) is that you don't have to use the extended syntax here.